packages/textlint-tester/src/test-util.ts

```diff
@@ -36,12 +36,14 @@ export type InvalidPatternArgs =
           textlint: TestTextlintLinter;
           inputPath: string;
           errors: TesterErrorDefinition[];
+          description: string | undefined;
```

Comment:
It looks like a missing fix.

Reply:
Sorry. Fixed [336439e](https://github.com/textlint/textlint/pull/1067/commits/336439e41e47bd55e8c998a6003a4c6b8bdc8cd8)